.circleci/config.yml

```diff
@@ -39,6 +39,8 @@ jobs:
     docker:
       - image: circleci/php:7.3
     working_directory: ~/elastic-apm-laravel
+    environment:
+      CC_TEST_REPORTER_ID: 23639f09f4ef515ddb5fea9baf90f9f5cd9bb5e0801cf1dc062d993a3d70d3eb
```

Comment:
[They have nice docs that answered my question](https://docs.codeclimate.com/docs/finding-your-test-coverage-token#should-i-keep-my-test-reporter-id-secret). Cool.